### DISCUSSION TITLE
Why can't I see all of my organizations?

### DISCUSSION THREAD

**Discussion by mike-loux-planview** [community/community#56400] [Mobile] [answered]
### Select Topic Area

Question

### Body

Maybe I'm especially thick, but when signed into the GitHub Mobile App with my work account (the one posting here), I can only see a single organization...

![image](https://github.com/community/community/assets/62146387/fa5acf66-dd81-48ac-a2fb-63b6af5b7be2)

...whereas on the web UI, I can see the 5 or so organizations I am a member of (including 2 that I am an owner of).  

![image](https://github.com/community/community/assets/62146387/2f078f88-ecb9-4654-8678-90733567fc47)

I cannot, for the life of me, figure out how to add or access the additional repositories, and it's frustrating the hell out of me.  Anybody else running into this issue and/or know what I need to do to fix this?  

At this point, I don't do much activity in the one org that I can see, so the mobile app is pretty useless except to approve sudo requests (and even that is annoying, because I can't pull up a list of requests on the app - I have to find the request in a list of notifications and approve it there).

Running the mobile app on a Samsung Galaxy S23 FE, v1.114.0

Thanks in advance for any wisdom anyone might have.

**Comment by eliperkins** [comment, marked as answer]
Hey @mike-loux-planview , thanks for raising this feedback! I know that there's some friction to this today, so let me see if I can help debug here

GitHub Mobile is an OAuth app, so as such, any given user authorizing GitHub Mobile will need to grant access to [organizations that have OAuth app restrictions](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps#oauth-apps-and-organizations) **at the time the user logs in**.

I'm wondering if it's possible that the organizations that you're trying to view don't appear here because the OAuth access token you're using with GitHub Mobile wasn't granted access to those organizations which might have OAuth restrictions.

Can you try the following?
1. Log out of the app in Profile > Settings > Manage Accounts
2. Log back into the app
3. During the login process, note which GitHub organizations appear within the OAuth flow under the "Organization Access" section when approving GitHub Mobile as an OAuth app

If these organizations _still_ don't appear within the app after completed those steps, let me know and I'll dive deeper with our engineering team here to figure out what's up!

**Reply by grishaLR** [reply]
came by this same issue in 2024 and this thread saved me! 

still relevant today 🫡

**Reply by daidv94** [reply]
Issue in 2025 after my org enforced Oauth with Okta. This thread saved my ass

**Comment by mike-loux-planview** [comment]
Hey, presto!  That did the trick!  
![image](https://github.com/community/community/assets/62146387/7e0e02be-f8ca-4f0a-90ee-2b32996b276d)
Hallelujah!  
Marked as answer.  Thank you!
